pkg/kubelet/api/v1alpha1/runtime/api.pb.go

```diff
@@ -230,17 +238,18 @@ type VersionResponse struct {
 	// The version of the kubelet runtime API.
 	Version *string `protobuf:"bytes,1,opt,name=version" json:"version,omitempty"`
 	// The name of the container runtime.
-	RuntimeName *string `protobuf:"bytes,2,opt,name=runtime_name" json:"runtime_name,omitempty"`
+	RuntimeName *string `protobuf:"bytes,2,opt,name=runtime_name,json=runtimeName" json:"runtime_name,omitempty"`
```

Comment:
This is all generated, it should probably be ripped out of this commit as it's overwritten in the other.